GitUI/Editor/FileViewerInternal.cs

```diff
@@ -137,12 +134,30 @@ public void SetText(string text, Action openWithDifftool, bool isDiff = false)
 
             // important to set after the text was changed
             // otherwise the may be rendering artifacts as noted in #5568
-            TextEditor.ShowLineNumbers = !isDiff;
+            TextEditor.ShowLineNumbers = ShowLineNumbers ?? !isDiff;
+            if (ShowLineNumbers.HasValue && !ShowLineNumbers.Value)
+            {
+                Padding = new Padding(DpiUtil.Scale(5), Padding.Top, Padding.Right, Padding.Bottom);
+            }
+
             TextEditor.Refresh();
 
             _currentViewPositionCache.Restore(isDiff);
         }
 
+        private static readonly SolidBrush PaddingBrush = new SolidBrush(Color.White);
```

Comment:
❗️ We can't do this. This will come and bite us when we tackle the theme.
The colour must be derived from the control.

Reply:
It was hard to find but I get it!
What do you prefer `new SolidBrush(TextEditor.ActiveTextAreaControl.TextArea.Document.HighlightingStrategy.GetColorFor("Default").BackgroundColor)` ( https://github.com/gitextensions/ICSharpCode.TextEditor/blob/9f443b2435241e4a528558af1b02d0691c285efc/Project/Src/Gui/TextView.cs#L238 ) which returns `SystemColor.Window` or directly `SystemColor.Window`?